README.md

```diff
@@ -1,3 +1,7 @@
+# Deprecation Warning!
+
+This package is **DEPRECATED** and should not receive any further updates except for security patches.  
```

Comment:
Shouldn't this go into `whisper/README.md`?

Reply:
oh yes, good catch

Reply:
🤣 Oh man! Thanks for spotting this.